--- psi-plus-0.15.4040/src/systeminfo.cpp~	2011-06-16 08:45:22.000000000 +0200
+++ psi-plus-0.15.4040/src/systeminfo.cpp	2011-06-16 08:49:40.974191785 +0200
@@ -108,7 +108,7 @@
 		QString file;
 		QString name;
 	} osInfo[] = {
-		{ LinuxCustom,		OsUseFile,	ApplicationInfo::homeDir() + "/custom-os",	""	},
+		{ LinuxCustom,		OsUseFile,	ApplicationInfo::homeDir(ApplicationInfo::ConfigLocation) + "/custom-os",	""	},
 
 		{ LinuxMandrake,	OsUseFile,	"/etc/mandrake-release",	"Mandrake Linux"	},
 		{ LinuxDebian,		OsAppendFile,	"/etc/debian_version",		"Debian GNU/Linux"	},
--- psi-plus-0.15.4127M/src/widgets/actionlineedit.h~	2011-09-09 20:10:15.000000000 +0200
+++ psi-plus-0.15.4127M/src/widgets/actionlineedit.h	2011-09-09 21:25:25.369838838 +0200
@@ -29,6 +29,8 @@
 
 class QMenu;
 
+QMenu *menu_;
+
 class ActionLineEditButton : public QAbstractButton
 {
 	Q_OBJECT
--- psi-plus-0.15.4127M/src/widgets/actionlineedit.cpp~	2011-09-09 20:10:15.000000000 +0200
+++ psi-plus-0.15.4127M/src/widgets/actionlineedit.cpp	2011-09-09 21:33:15.916458410 +0200
@@ -226,15 +226,3 @@
 	menu->exec(e->globalPos());
 	delete menu;
 }
- 
-void ActionLineEdit::contextMenuEvent(QContextMenuEvent *e)
-{
-	QMenu *menu;
- 	if (isReadOnly()) {
-		menu_->exec(e->globalPos());
-		e->accept();
- 	}
- 	else {
-		QLineEdit::contextMenuEvent(e);
- 	}
-}
